students/JerryH/Lesson08/circle.py

```diff
@@ -0,0 +1,57 @@
+import math
+
+class Circle:
+
+    def __init__(self, radius):
+        self.radius = radius
+        self._area = radius ** 2 * math.pi
+
+    @property
+    def diameter(self):
+        return self.radius * 2
+
+    @diameter.setter
+    def diameter(self, value):
+        self.radius = value / 2
+
+    @property
+    def area(self):
+        """ Read Only """
+        return self._area
+
+    @property
+    def from_diameter(self):
+        return self.diameter
+
+    #@from_diameter.setter
+    def from_diameter(self, diameter):
+        self.diameter = diameter
+
+    def __repr__(self):
+        return "Circle({})".format(self.radius)
+
+    def __str__(self):
+        return "Circle with radius: {}".format(self.radius)
+
+    def __add__(self, other):
+        return "Circle({})".format(self.radius + other.radius)
+
+    def __mul__(self, other):
+        return "Circle({})".format(self.radius * other)
```

Comment:
__add__ and __mul__ methods should do actual addition and multiplication, not return the string repr of it. (the assignment output on the website is produced by the `__repr__` function)